packages/web-app-files/tests/unit/components/SideBar/SideBar.spec.ts

```diff
@@ -75,14 +79,46 @@ describe('SideBar', () => {
       })
     })
   })
+  describe('tags panel', () => {
```

Comment:
```suggestion
  describe('tags menu item', () => {
```

Reply:
`tags panel` is pretty fitting IMO. We're basically testing if the panel from `fileSideBars.ts` is being loaded or not.